### ISSUE TITLE
Обнаружение, подключение и управления дистанционно управляемыми вибромассажерами.

### ISSUE THREAD

**Issue by MikeRebel** [flipperdevices/flipper-hackathon-moscow#23]
### Подробно опишите идею вашего проекта

Сделать функционал по управлению с устройства дистанционно управляемыми вибромассажерами. 

### Готовый проект будет уметь...

Можно подключить вибромассажер, увеличить и уменьшить силу вибрации, переключить режим.

### Вы готовы приехать на Хакатон в Москву?

- [X] Yes, my team will attend

---

# Капитаном назначен @vvzvlad

**Comment by zhovner** [comment]
Все нормальные BLE устройства используют криптографию. Вы реализовывали подобно уже?

**Comment by MikeRebel** [comment]
Немного непонятно, почему вы решили, что это проблема? Я не про взлом устройства, а про управление им хочу сделать ПО. Опыта нет

**Comment by anna-oake** [comment]
Да, протоколы многих игрушек отреверсены, и при желании их можно реализовать на Флиппере.
Только имейте в виду, что наш BLE пока не имеет удобных высокоуровневых API для взаимодействия с другими устройствами, кроме собственного протокола RPC.

**Comment by vvzvlad** [comment]
@MikeRebel Можно в команду?

**Comment by MikeRebel** [comment]
Конечно

**Comment by vvzvlad** [comment]
@aprosvetova что надо сделать для добавления участника? Просто отредактировать первый пост?
UPD: вижу, уже отредактирован

**Comment by anna-oake** [comment]
### Ждём ваше финальное решение

**Если вы не идёте на Хакатон**, напишите комментарий здесь и закройте issue.

**Если изменилось число участников**, напишите здесь, кого нужно исключить.

**Если вы ТОЧНО идёте на Хакатон**, каждый участник команды должен:
1. Зарегистрироваться на мероприятие [по ссылке](https://neuronspace.timepad.ru/event/1860939/)
2. Сообщить здесь, в issue, о прохождении регистрации

**Comment by vvzvlad** [comment]
Зарегистрировался.

**Comment by MikeRebel** [comment]
Привет. 
Я не успеваю поучаствовать, от нас будет 1 человек.

**Comment by anna-oake** [comment]
Договорились, спасибо!
